### ISSUE TITLE
Prototyping S/CM and RAM working as team in DSH

### ISSUE THREAD

**Issue by malvikasharan** [alan-turing-institute/open-community-building#201]
### Summary Sentence

This year we will have many projects where S/CMs and RAMs are recruited together (DCE, DT, DSH)
Previously these roles have worked separately, and we will have consider how they complement each other work.

### What needs to be done?

Jen and I had a meeting to discuss this, with Gabin and Chris in attendance. I had a chance to touch base with Arron and he is onboard. 💪🏽 

Arron will help us prototype this in DSH by utilising resources ([listed in sheet 1](https://docs.google.com/spreadsheets/d/1pegEKZe0l6txwvJi1w8BB2OMzH3ov7tASH8ziC8xg5k/edit#gid=1714497170)) as well as documenting how he aligns his workflow and processes with Hari and RAM's processes, where confusions and challenges exist and what new S/CM - RAM teams should consider.

### Who can help?

@arronlacey @dingaaling  @harisood 

---

### Update after the issue was opened

[Add details]

**Comment by Davsarper** [comment]
Yes please, this would be great to discuss and be there for. DSH is a great pilot case for all of us research infrastructure role to cohabit

**Comment by malvikasharan** [comment]
@dingaling @GabinWK and I have been working on a RAM-RCM combined stakeholder document. I will ping the work in progress with you privately - the documents is not public yet.
